packages/@react-spectrum/avatar/docs/Avatar.mdx

```diff
@@ -0,0 +1,80 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/avatar';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/avatar/package.json';
+
+```jsx import
+import {Avatar} from '@react-spectrum/avatar';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Navigation
+keywords: [avatar]
+after_version: 3.0.0
+---
+
+# Avatar
+
+<PageDescription>{docs.exports.Avatar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Avatar']}
+  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/avatar/'}]} />
+
+## Example
+
+```tsx example
+<Avatar src="https://mir-s3-cdn-cf.behance.net/project_modules/disp/690bc6105945313.5f84bfc9de488.png" alt="default Adobe avatar" />
```

Comment:
Not a huge deal, but maybe we could use a shorter url. We could link this via imgur.com, like we have on the Image docs. It also seems like then the image would be less likely to disappear?

Reply:
right, I meant to ask if the URL should be imgur or behance.